### PR TITLE
CNF-10170: bootkube.sh: Render kernel boot arguments for SNO

### PR DESCRIPTION
**Pull request by jlojosnegros**
In an effort to speed up installation and avoid node reboots this command will run `tuneD` on a properly isolated environment and render a MachineConfig to apply the kernel boot args readed from `/etc/tuned/bootcmdline`

reverts: #7979 

See: 
- https://github.com/openshift/cluster-node-tuning-operator/pull/935
- https://github.com/openshift/cluster-node-tuning-operator/pull/844

### CODE REVIEW

**Comment by jlojosnegros** [comment]
/hold 
until https://github.com/openshift/cluster-node-tuning-operator/pull/935 is merged

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 1]
```diff
@@ -348,6 +348,44 @@ then
...
+if [ "$BOOTSTRAP_INPLACE" = true ] && [ ! -f cmdbootline-nto-bootstrap.done ]; then
+	record_service_stage_start "cmdbootline-nto-bootstrap"
+
+	rm --recursive --force cmdbootline-nto-bootstrap
+
+	# Create folders for overlay and podman command to use them
+	tmpfolder=$(mktemp -d)
+	echo ${tmpfolder}
+	podmancmd="bootkube_podman_run --attach=stderr --rm "
+	for folder in var/lib/tuned/profile-data etc/modprobe.d etc/sysconfig etc/kubernetes etc/sysctl.d etc/systemd run sys lib/modules var/lib/kubelet usr/lib/tuned etc/tuned; do
```
Comment (yanirq) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:360]: @jmencak  @jlojosnegros   do we need to align with https://github.com/openshift/cluster-node-tuning-operator/pull/970  in some manner ? or that change is not affecting the installer ?

Comment (jmencak) [reply data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:360]: Good point. It looks like it needs an update. Also note that `/var/lib/tuned/profiles-data` was removed, not sure what `var/lib/tuned/profile-data` (singular) is used for. @jlojosnegros typo?

Comment (jlojosnegros) [reply data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:360]: yeah it is a typo

**Comment by vitus133** [comment]
/close
continue this work at #8099

**Pull request closed without merging**